src/layout/DwindleLayout.cpp

```diff
@@ -818,9 +818,18 @@ void CHyprDwindleLayout::toggleWindowGroup(CWindow* pWindow) {
 
             PWINDOW->m_bHidden = false;
         }
+	    
+	if(PHEAD->pPreviousGroupMember)
+	{
```

Comment:
style: indent is 4 and braces are not on newlines

Reply:
done?

Reply:
ugh

Reply:
what?

Reply:
fixed it for you. Why cant you just look around how code looks like..

Reply:
well thanks, i guess.
I will try harder next time!